packages/core/test/unit/lib/nunjucks-extensions/set-external.test.js

```diff
@@ -0,0 +1,42 @@
+const nunjucks = require('nunjucks');
+const path = require('path');
+const { SetExternalExtension } = require('../../../../src/lib/nunjucks-extensions/set-external');
+
+test('does not render the header row when noHeader is not set', () => {
```

Comment:
Thanks for creating these as unit tests!

I was actually just expecting [functional site tests](https://markbind.org/devdocs/devGuide/workflow.html#updating-and-writing-tests). They don't require much maintenance, and test for unexpected diffs as well.

But this seems better 👍